tango_simlib/tests/test_tango_sim_generator.py

```diff
@@ -18,63 +18,131 @@
 
 from tango_simlib import tango_sim_generator
 from tango_simlib.tests import test_sim_test_interface
-from tango_simlib.utilities import helper_module, sim_xmi_parser, fandango_json_parser
+from tango_simlib.utilities import (helper_module, sim_xmi_parser,
+                                    fandango_json_parser, simdd_json_parser)
 from tango_simlib.utilities.testutils import ClassCleanupUnittestMixin
 
 MODULE_LOGGER = logging.getLogger(__name__)
 
+class BaseTest:
```

Comment:
Should inherit from object.  `BaseTest(object)`

Reply:
Sure.